### DISCUSSION TITLE
Scratch Addons on iPad

### DISCUSSION THREAD

**Discussion by ghost** [ScratchAddons/ScratchAddons#5118] [Help]
Is it possible to get scratch addons on the iPad? If I can, then how?

**Comment by rgantzos** [comment]
Unfortunately, it isn’t at the moment because Scratch Addons isn’t available as a Safari extension. However, multiple Android browsers support Scratch Addons.

**Comment by BroJac5246** [comment]
You would need a browser that supports extensions on ipad, which could be hard to find.

**Reply by rgantzos** [reply]
There aren’t any.

**Reply by BroJac5246** [reply]
> There aren’t any.

Could you get any through a jailbreak (theoretically)?

**Reply by Ksplover2343** [reply]
Safari supports extensions, I am sure that is all though.

**Comment by WorldLanguages** [comment]
TurboWarp supports most editor addons.
